web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
[I'm most definitely using this](https://github.com/laslabs/web/blob/0b624b5c783e7db137222ba0c1fe9cf7edb7a47a/web_app_drawer/views/web.xml#L139) - Bootstrap doesn't support submenus. They were removed before 3.0 release IIRC

I could instead do what Enterprise does, and put everything in one menu. I'm admittedly torn on whether I like the secondary dropdown levels or not.